package.json

```diff
@@ -36,8 +36,8 @@
     "standard": "^10.0.2"
   },
   "engines": {
-    "node": ">= 0.8.x",
-    "npm": ">= 1.1.x"
+    "node": ">= 4.8.3",
```

Comment:
yes it’s the last maintenance release. But I think you are right, we should probably be fine with `>=4` and npm `>=2`, will show less errors. And if people run into problems with `4.x` versions less than 4.8 then we can bump that, I’ll update the PR